File: src/main/java/ru/sicampus/bootcamp2026/repository/MeetingRepository.java
package ru.sicampus.bootcamp2026.repository;

import org.springframework.data.domain.Page;
import org.springframework.data.jpa.repository.JpaRepository;
import ru.sicampus.bootcamp2026.entity.Meeting;
import ru.sicampus.bootcamp2026.entity.User;

import java.util.List;
import java.util.Optional;

public interface MeetingRepository extends JpaRepository<Meeting, Long> {
    List<Meeting> findAllByTitle(String title);

    List<Meeting> findAllByCreator(User user);

    //Page<Meeting> findBy
}

File: src/main/kotlin/ru/sicampus/bootcamp2026/repository/MeetingRepository.kt
package ru.sicampus.bootcamp2026.repository

import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository
import ru.sicampus.bootcamp2026.entity.Meeting
import java.time.LocalDate

@Repository
interface MeetingRepository : JpaRepository<Meeting, Long> {
    fun findByOrganizerId(organizerId: Long): List<Meeting>
    fun findByDate(date: LocalDate): List<Meeting>
}
